docs/setup/environment-vars.mdx

```diff
@@ -23,6 +23,22 @@ MindsDB does not require authentication by default. If you want to enable authen
     ```
 </CodeGroup>
 
+## MindsDB Configuration File
+
+In order to start MindsDB with a [custom configuration file](/setup/custom-config), the `MINDSDB_CONFIG_PATH` environment variable should store the file path.
+
+### Example
+
+<CodeGroup>
+    ```bash Docker
+    docker run --name mindsdb_container -e MINDSDB_CONFIG_PATH=/Users/username/path/config.json -e MINDSDB_APIS=http,mysql -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
+    ```
+
+    ```bash Shell
+    export MINDSDB_CONFIG_PATH=/Users/username/path/config.json
```

Comment:
may be add as next line command that run mindsdb? `python -m mindsdb --api=http,mysql`